src/sdk/lsf-sdk.js

```diff
@@ -587,7 +592,12 @@ export class LSFWrapper {
         {
           body: serializedAnnotation,
         },
-      );
+      ).then(result => {
+        const status = result?.$meta?.status;
+
+        if (status === 200 || status === 201) return this.datamanager.invoke("toast", { message: "Annotation updated successfully", type: "info" });
+        else if (status !== undefined) return this.datamanager.invoke("toast", { message: "There was an error updating your Annotation", type: "error" });
+      });
```

Comment:
```suggestion
        const status = result?.$meta?.status;

        if (status === 200 || status === 201) this.datamanager.invoke("toast", { message: "Annotation updated successfully", type: "info" });
        else if (status !== undefined) this.datamanager.invoke("toast", { message: "There was an error updating your Annotation", type: "error" });
        return result;
      });
```

Reply:
The result if not returned from the inner promise will yield undefined in the outer result usage.